pkgs/tools/networking/network-manager/fix-paths.patch

```diff
@@ -63,4 +65,3 @@
  	g_ptr_array_add (argv, "--use-blacklist");
  	g_ptr_array_add (argv, (char *) arg1);
  
- 	va_start (ap, arg1);
```

Comment:
is this an accidental removal?

Reply:
Ah, this caught my eye too. But then I realized that this is a line removed from the exiting patch, rather than a new diff within this patch dictating that `va_start ...` should be removed from the target file -- so as long as this patch applies, the patch is semantically the same as before.

Reply:
Oh I see what you mean now, man that's confusing! Thanks for pointing that out :man_facepalming: